rest.exceptions.badrequest=Bad request
rest.exceptions.conflict=Conflict
rest.exceptions.internalerror=Internal error
rest.exceptions.notfound=Not found
rest.exceptions.unauthorized=Unauthorized

rest.response.resource.created=Resource created successfully
rest.response.resource.updated=Resource updated successfully
rest.response.resource.deleted=Resource with id {0} updated successfully

rest.response.suborders.preparing=Suborder {0} was marked as preparing
rest.response.suborders.postponed=Suborder {0} was marked as postponed
rest.response.suborders.finished=Suborder {0} was marked as finished
rest.response.orders.cancelled=Order {0} was cancelled
rest.response.orders.updated=Order {0} was successfully updated
rest.response.orders.closed=Order {0} was closed

notifications.payment.success=Payment success for order {0}
notifications.payment.fail=Payment for order {0} failed. Reason: {1}
notifications.restaurant.suborder.available=New order available.\nID: {0}\n{1} items:\n{2}.\nTotal: {3}.
notifications.order.created=Your order {0} has been taken into work. We will start preparing it soon.
notifications.order.ready=Your order {0} is ready. Thank you for using our service!
notifications.order.cancelled=Your order {0} was cancelled
notifications.order.postponed=Your order {0} was postponed by the restaurant
notifications.order.preparing=The restaurant is preparing your order {0}
notifications.payment.error.expired=Payment expired

email.subjects.payment=Order payment status
email.subjects.order=Your order status
email.subjects.suborder=New suborder available

exceptions.conflict.product=Product with id {0} already exists
exceptions.conflict.restaurant=Restaurant with id {0} already exists
exceptions.notfound.suborder=Suborder with id {0} not found

